upgrade-notes/storefront_20241127_183646.md

```diff
@@ -0,0 +1,3 @@
+#### Added mappers for Convertim payment and transport types ([#3624](https://github.com/shopsys/shopsys/pull/3624))
+
+- see #project-base-diff to update your project
```

Comment:
I would emphasize the need to ignore payments and transports that should not be displayed in the cart.